apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```diff
@@ -272,6 +272,8 @@ update.local.user.claims.error=Error occurred while updating claims for local us
 retrieving.realm.to.handle.claims.error=Error occurred while retrieving the Realm for the tenant domain to handle local claims.
 post.auth.cookie.not.found=Your authentication flow is ended or invalid. Post authentication sequence tracking cookie not found in the request. Please initiate again.
 
+authentication.failed=Authentication Failed !
```

Comment:
`Authentication Failed!` Remove the space before `!`

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3162